### PR TITLE
ADCC GTL & Luciferase template

### PR DESCRIPTION
**Pull request by ruizhw19**
Hi Alicia and Bhavesh, I added biological endpoints, lab methods, statistical methods templates for ADCC GTL & Luciferase & ICABA ready for review:
- [GTL](https://github.com/FredHutch/VISCtemplates/tree/ADCC-template/inst/templates/methods-adcc/GTL)
- [Luciferase](https://github.com/FredHutch/VISCtemplates/tree/ADCC-template/inst/templates/methods-adcc/Luciferase)
- [ICABA](https://github.com/FredHutch/VISCtemplates/tree/ADCC-template/inst/templates/methods-adcc/ICABA)

### CODE REVIEW

File: inst/templates/methods-adcc/Luciferase/adcc-luc-lab-methods.Rmd [hunk 1]
```diff
@@ -8,10 +8,11 @@ date: '2022-10-28'
...
 
 Target and effector cells were plated in white 96-well half-area plates and co-cultured with 4-fold [Adjust if different] serial dilutions of trial participant serum starting at the 1:50 dilution. For each sample, percent specific killing was measured in duplicate at dilutions of 1:50, 1:200, 1:800, 1:3200, 1:12800, and 1:51200 [Adjust if dilutions are different]. Co-cultures were incubated for 6 hours at $37 ^{\circ} C$ in $5\% \text{ CO}_2$. The final readout was the reduction of luminescence intensity generated by the presence of residual intact target cells that had not been lysed by the effector population in the presence of ADCC-mediating serum antibodies. The percentage of killing was calculated using the formula: percent specific loss Luciferase activity = 100 * (RLU of target and effector well– RLU of test well)/(RLU of target and effector well).
```
Comment (brborate) [inline inst/templates/methods-adcc/Luciferase/adcc-luc-lab-methods.Rmd:15]: Target and effector cells were plated in white 96-well half-area plates and co-cultured with trial participant serum sample starting at the 1:50 dilution and 4-fold serial dilutions thereon (1:200, 1:800, 1:3200, 1:12800, and 1:51200) [Adjust if dilutions are different]. Co-cultures were incubated for 6 hours at $37 ^{\circ} C$ in $5\% \text{ CO}_2$. For each serial dilution of the sample, the final readout was measured in duplicate. The final readout was the reduction of luminescence intensity generated by the presence of residual intact target cells that had not been lysed by the effector population in the presence of ADCC-mediating serum antibodies. The percentage of killing was calculated using the formula: percent specific loss Luciferase activity = 100 * (RLU of target and effector well– RLU of test well)/(RLU of target and effector well).

Comment (brborate) [reply inst/templates/methods-adcc/Luciferase/adcc-luc-lab-methods.Rmd:15]: Additional comment: 
It is not clear from the text how the duplicates were performed. My understanding is that each sample is split into two, and each of the two splits are serially diluted and final readouts taken. If that is correct, it should help if we mention that explicitly.

File: inst/templates/methods-adcc/Luciferase/adcc-luc-lab-methods.Rmd [hunk 2]
```diff
@@ -8,10 +8,11 @@ date: '2022-10-28'
 
 
 # Lab Methods
-CEM.NKR~CCR5~ cells [@trkola1999cell] were 
-used as targets for ADCC luciferase assays after infection with the infectious molecular clones ([insert IMCs]). 
+CEM.NKR~CCR5~ cells [@trkola1999cell] were used as targets for ADCC luciferase assays after infection with the infectious molecular clones ([insert IMCs]).
```
Comment (brborate) [inline inst/templates/methods-adcc/Luciferase/adcc-luc-lab-methods.Rmd:11]: For the ADCC Luciferase assay, the target cells need to be transfected with the luciferase reporter gene as well. Suggest the following wording (to be confirmed by the lab):

CEM.NKR~CCR5~ cells [@trkola1999cell] transfected with the luciferase reporter gene and infected with the infectious molecular clones ([insert IMCs]) were used as targets for ADCC luciferase assays.

**Review by brborate — DISMISSED** [review]
Hi Cassidy, 
Thanks for working on this. This is great work. I made a few minor comments. 
Best, 
Bhavesh

**Comment by lemireg** [comment]
Hello team @asatofh @slager @mayerbry,

Updated to incorporate most recent ADCC template code aligning with HVTN per:
`T:\vaccine\documents\templates\lab_assay_memos\ADCC-Flow_Memo_PT_Template_LS.docx`
`T:\vaccine\documents\templates\lab_assay_memos\ADCC-Luc_Memo_PT_Template_2022-12-21.docx`

These files reflects the above HVTN templates (and additional feedback Shannon had with lab in using % specific killing instead of % loss luciferase):
`inst/templates/methods-adcc/adcc-lab-methods.Rmd`
`inst/templates/methods-adcc/adcc-biological-endpoints.Rmd`
`inst/templates/methods-adcc/adcc-statistical-methods.Rmd`
`inst/templates/bibliography.bib`

Other files are modified to allow loading in of report_type = "adcc":
`R/use_visc_report.R`
`man/use_visc_methods.Rd`
`man/use_visc_report.Rd`

I have loaded in this branch, generated the report with `use_visc_report(report_type = "adcc")` and knit the file and it knits as expected. See attached PDF.
[TEST_adcc.pdf](https://github.com/user-attachments/files/16463695/TEST_adcc.pdf)

It should be ready for merge!

**Review by mayerbry — APPROVED** [review]
I went through and made a test adcc report locally, looks good and clean from my end!

File: inst/templates/methods-adcc/adcc-lab-methods.Rmd [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+ADCC is quantified as net percent granzyme B activity, which is the percent of target cells positive for GTL (an indicator of granzyme B uptake) minus the percent of target cells positive for GTL when incubated with effector cells in the absence of a source of antibodies. Flow cytometry is used to quantify the frequency of granzyme B positive cells.
+
+# Lab Methods Luciferase
+
+We utilized a modified version of a previously published ADCC luciferase procedure [@Pollara2014, @Fisher2019] . Briefly, CEM.NKRCCR5 cells [@Trkola1999] were used as targets for ADCC luciferase assays after infection by one of the following HIV-1 [vaccine-matched, if all IMCs are vaccine-matched; if not, indicate match in table below] IMCs:
```
Comment (slager) [inline inst/templates/methods-adcc/adcc-lab-methods.Rmd:17]: separate multiple references in same citation with a semicolon, not comma

**Review by slager — CHANGES_REQUESTED** [review]

**Comment by slager** [comment]
Looks great! Knits cleanly on CI! Just 1 minor comment about a reference.

File: inst/templates/methods-adcc/adcc-lab-methods.Rmd (hunk 3 above)
Comment (lemireg) [reply inst/templates/methods-adcc/adcc-lab-methods.Rmd]: Great catch! I've updated in commit 90fa50e

**Review by slager — APPROVED** [review]

**Review by brborate — CHANGES_REQUESTED** [review]

File: inst/templates/methods-adcc/adcc-statistical-methods.Rmd [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+Baseline-subtracted peak percent (\%) specific killing was defined as the maximum baseline-subtracted activity across the six dilution levels.
+
+### AUC
+
+Nonparametric partial Area under the baseline-subtracted curves ("pAUC"), calculated using the trapezoidal rule on three dilutions of the baseline-subtracted curves, setting baseline-subtracted percent specific killing less than 0\% to zero.
+
+### Response call
+
+A response is defined as positive if the peak baseline-subtracted \% specific killing activity greater than or equal to 10\% for either the 1:50 or 1:200 dilution.
```
[the remark this answers is not in the document]
Comment (lemireg) [reply inst/templates/methods-adcc/adcc-statistical-methods.Rmd:38]: I'm pretty sure this correction is right (i.e. drop the "peak" from the sentence in the original text. I have incorporated that feedback. Will double check with @asatofh

File: inst/templates/methods-adcc/adcc-lab-methods.Rmd [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+<!-- This file was adapted from VISCtemplates {{pkg_ver}}. -->
+
+
+# Lab Methods GTL
+
+The qualified GranToxiLux Antibody-Dependent Cell-Mediated Cytotoxicity (GTL-ADCC) assay was performed as previously described [@Pollara2014]. Target cells were a clonal isolate of the CEM.NKRCCR5 CD4+ T-cell line (NIH AIDS Reagent Program, Division of AIDS, NIAID, NIH: from Dr. Alexandra Trkola [@Trkola1999]. These cells were  coated with recombinant gp120s representing the HIV-1 envelopes of the subtype [specify subtype and antigens, e.g. C (TV1 and 1086c)]. Effector cells were PBMCs obtained from a HIV-seronegative donor with heterozygous for Fc$\gamma$R3A at position 158 (158F/V). PBMCs were obtained by leukapheresis to collect enough cells for completion of the study with a single donation, minimizing potential effector cell population variability effects on the study outcome.  PBMCs were used at an effector cell to target cell ratio of 30:1. Serum samples were tested after five-fold serial dilutions starting at 1:50 (1:50, 1:250, 1:1250, 1:6250, 1:31250, and 1:156250). Each plate has one standardized positive control in duplicate and one standardized negative control in duplicate.
```
[the remark this answers is not in the document]
Comment (lemireg) [reply inst/templates/methods-adcc/adcc-lab-methods.Rmd:10]: Double check second comment with @asatofh

File: inst/templates/methods-adcc/adcc-statistical-methods.Rmd [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+# Statistical Methods GTL and Luciferase
+
+## Graphical analysis
+
+Plots of peak activity and AUC show both response rates and the distribution of magnitude. Positive responses are indicated by dots color-coded by treatment group, and negative responses by gray triangles. A boxplot is superimposed on the distribution, including only positive responses. The mid-line of the box denotes the median and the ends of the box denote the $25^{th}$ and $75^{th}$ percentiles. The whiskers that extend from the top and bottom of the box extend to the most extreme data points that are no more than 1.5 times the interquartile range (i.e., height of the box) or if no value meets this criterion, to the data extremes.
```
[the remark this answers is not in the document]
Comment (lemireg) [reply inst/templates/methods-adcc/adcc-statistical-methods.Rmd:44]: A part of a wider conversation: discuss with @asatofh (and the rest of the team). Do we want to add RR to the plot in the template, or remove the reference to that plot here?

File: inst/templates/methods-adcc/adcc-statistical-methods.Rmd [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+### AUC
+
+Nonparametric partial Area under the baseline-subtracted curves ("pAUC"), calculated using the trapezoidal rule on three dilutions of the baseline-subtracted curves, setting baseline-subtracted percent specific killing less than 0\% to zero.
+
+### Response call
+
+A response is defined as positive if the peak baseline-subtracted \% specific killing activity greater than or equal to 10\% for either the 1:50 or 1:200 dilution.
+
+# Statistical Methods GTL and Luciferase
```
[the remark this answers is not in the document]
Comment (lemireg) [reply inst/templates/methods-adcc/adcc-statistical-methods.Rmd:40]: Check with @asatofh. The BAMA, NAb, and generic reports are all in past tense, so I imagine we'll change this to past tense. Thanks for noticing Bhavesh!

File: inst/templates/methods-adcc/adcc-lab-methods.Rmd (hunk 5 above)
[the remark this answers is not in the document]
Comment (lemireg) [reply inst/templates/methods-adcc/adcc-lab-methods.Rmd]: Checking in with the lab about this. Thanks for the comment @brborate !

File: inst/templates/methods-adcc/adcc-statistical-methods.Rmd (hunk 4 above)
[the remark this answers is not in the document]
Comment (lemireg) [reply inst/templates/methods-adcc/adcc-statistical-methods.Rmd]: Also following up with lab on this.

File: inst/templates/methods-adcc/adcc-statistical-methods.Rmd (hunk 6 above)
[the remark this answers is not in the document]
Comment (lemireg) [reply inst/templates/methods-adcc/adcc-statistical-methods.Rmd]: Per a wider conversation we are going to be deleting demo figures from templates (including the figure mentioned in your comment @brborate), but we want to keep template example wording. So this section will remain the same as a sample, and the analyst will have to revise as needed for their use case. We will add a comment in the Rmd "please delete or modify per the specs of your report"

File: inst/templates/methods-adcc/adcc-statistical-methods.Rmd (hunk 7 above)
[the remark this answers is not in the document]
Comment (lemireg) [reply inst/templates/methods-adcc/adcc-statistical-methods.Rmd:40]: Will change to : "A response **was** defined as positive if the peak baseline-subtracted \% specific killing activity greater than or equal to 10\% for either the 1:50 or 1:200 dilution."

We are also following up on whether to keep peak or not in this sentence.

**Review by brborate — APPROVED** [review]
Approving this PR since we still are waiting to hear back from lab (as per discussion with Gabby).

**Pull request merged**